src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs

```diff
@@ -85,7 +86,7 @@ void M()
 
             var results = await RunGetCodeActionsAsync(testLspServer, caretLocation);
             var introduceConstant = results[0].Children.FirstOrDefault(
-                r => ((CodeActionResolveData)r.Data).UniqueIdentifier == FeaturesResources.Introduce_constant
+                r => ((JObject)r.Data).ToObject<CodeActionResolveData>().UniqueIdentifier == FeaturesResources.Introduce_constant
```

Comment:
data is defined as an object in the CodeAction, so streamjsonrpc will only deserialize to jobject.  Before no deserialization was happening so we could just cast.